### PR TITLE
Some updates for efficiency QA plots

### PR DESCRIPTION
**Pull request by zhangbiao-phy**

### CODE REVIEW

**Review by vkucera — COMMENTED** [review]
Where is the function `efficiencyhadron` called?

File: FirstAnalysis/efficiency_studies.py [hunk 1]
```diff
@@ -19,36 +27,68 @@ def efficiencytracking(var):
...
+        for j in range(0, gr.GetN()):
+            gr.GetEXlow()[j] = 0
+            gr.GetEXhigh()[j] = 0
+
+        gr.SetLineColor(color_list[i])
+        gr.SetMarkerColor(color_list[i])
+        gr.SetMarkerStyle(marker_list[i])
+        eff_list.append(gr)
+        gr.Draw(" same p")
+        leg.AddEntry(eff_list[i], had, "p")
```
[the remark this answers is not in the document]
Comment (zhangbiao-phy) [reply FirstAnalysis/efficiency_studies.py:91]: > What is the use of `eff_list`?

It's used to collect the marker with species.

**Comment by zhangbiao-phy** [comment]
> Where is the function `efficiencyhadron` called?

I can add them now.

File: FirstAnalysis/efficiency_studies.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (vkucera) [reply FirstAnalysis/efficiency_studies.py:91]: > > What is the use of `eff_list`?
> 
> It's used to collect the marker with species.

Do you mean that `gr` is deleted from memory at the next iteration of the for loop if it is not kept in `eff_list`?

[the remark this answers is not in the document]
Comment (zhangbiao-phy) [reply FirstAnalysis/efficiency_studies.py:91]: > > > What is the use of `eff_list`?
> > 
> > 
> > It's used to collect the marker with species.
> 
> Do you mean that `gr` is deleted from memory at the next iteration of the for loop if it is not kept in `eff_list`?

yes, That's why I use this group

**Review by vkucera — APPROVED** [review]

**Pull request merged**